# Generated by Django 3.2.16 on 2023-04-04 09:56

import datetime
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('app1', '0017_auto_20230403_0437'),
    ]

    operations = [
        migrations.CreateModel(
            name='Medibot_Diagnosis_Dec',
            fields=[
                ('user', models.CharField(blank=True, max_length=1500, null=True)),
                ('id', models.BigAutoField(primary_key=True, serialize=False)),
                ('disease', models.CharField(blank=True, max_length=1500, null=True)),
                ('module_name', models.CharField(blank=True, max_length=150, null=True)),
                ('dateandtime', models.DateTimeField(blank=True)),
            ],
        ),
        migrations.CreateModel(
            name='Medibot_modules_details',
            fields=[
                ('uniqueis', models.CharField(blank=True, max_length=1500)),
                ('user', models.CharField(blank=True, max_length=1500, null=True)),
                ('id', models.BigAutoField(primary_key=True, serialize=False)),
                ('Qn', models.CharField(blank=True, max_length=150, null=True)),
                ('Ans', models.CharField(blank=True, max_length=150, null=True)),
                ('module_name', models.CharField(blank=True, max_length=150, null=True)),
                ('dateandtime', models.DateTimeField(blank=True)),
            ],
        ),
        migrations.AlterField(
            model_name='feedbackupdated',
            name='dateandtime',
            field=models.DateTimeField(blank=True, default=datetime.datetime(2023, 4, 4, 9, 56, 27, 585486)),
        ),
    ]
